core/src/util/jobQueue.cpp

```diff
@@ -0,0 +1,40 @@
+#include "jobQueue.h"
+#include <mutex>
+
+namespace Tangram {
+
+std::unordered_map<JobQueue::ThreadId, std::vector<JobQueue::Job>> JobQueue::jobsForThreads;
+std::mutex jobMutex;
```

Comment:
Oh you make a fair point - I thought that neither of those were class members. I've edited this such that both variables are now non-members.